tests/test_create_command_docs.py

```diff
@@ -0,0 +1,76 @@
+import os
+from unittest import TestCase
+
+from click.testing import CliRunner
+
+from tests.conftest import BASE_DIR
+from utils.create_command_docs import docs
+
+
+class TestCreateCommandDocsCli(TestCase):
+    def setUp(self) -> None:
+        self.runner = CliRunner()
+
+    @classmethod
+    def tearDownClass(cls):
+        os.remove(f"{BASE_DIR}/tests/fixtures/docs.md")
+
+    def test_check_required_module_option(self):
+        result = self.runner.invoke(docs)
+
+        output = result.output
+
+        assert result.exit_code != 0
+        assert "Error: Missing option '--module' / '-m'." in output
+
+    def test_check_required_cmd_option(self):
+        result = self.runner.invoke(docs, ["--module", "foo"])
+
+        output = result.output
+
+        assert result.exit_code != 0
+        assert "Error: Missing option '--cmd' / '-c'." in output
+
+    def test_check_required_output_option(self):
+        result = self.runner.invoke(docs, ["--module", "foo", "--cmd", "bar"])
+
+        output = result.output
+
+        assert result.exit_code != 0
+        assert "Error: Missing option '--output' / '-o'." in output
+
+    def test_check_valid_module_option(self):
+        result = self.runner.invoke(docs, ["--module", "foo", "--cmd", "bar", "--output", "baz"])
+
+        output = result.output
+
+        assert result.exit_code != 0
+        assert "Could not find module: foo. Error: No module named 'foo'" in output
+
+    def test_check_valid_cmd_option(self):
+        result = self.runner.invoke(docs, ["--module", "copilot_helper", "--cmd", "bar", "--output", "baz"])
+
+        output = result.output
+
+        assert result.exit_code != 0
+        assert "Error: Could not find command bar in copilot_helper module" in output
+
+    def test_create_command_docs(self):
+        output_path = f"{BASE_DIR}/tests/fixtures/docs.md"
```

Comment:
Maybe delete this file and assert it does not exist as part of the arrange.

Reply:
The file is always deleted at the end of the test suite. I could add the assertion though to check that the file does not exist yet - will add this in a subsequent commit.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/83/commits/225e94c3b9eb6d7be3b7f81b03df9903b43bd707.